src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * This class contains functionality that deals with making sense of user input
```

Comment:
I think the javadoc comments can be placed right before the class declaration instead of at the start.